h5py/tests/test_file.py

```diff
@@ -138,6 +138,30 @@ def test_invalid_mode(self):
         with self.assertRaises(ValueError):
             File(self.mktemp(), 'mongoose')
 
+@ut.skipIf(h5py.version.hdf5_version_tuple < (1, 10, 1),
+               'Requires HDF5 1.10.1 or later')
+class TestSpaceStrategy(TestCase):
+
+    """
+        Feature: Create file with specified file space strategy
+    """
+
+    def test_create_with_space_strategy(self):
+        """ Create file with file space strategy """
+        fname = self.mktemp()
+        fid = File(fname, 'w', fs_strategy="page",
+                   fs_persist=True, fs_threshold=100)
```

Comment:
Check retrieving the values as well? `fid.id.get_create_plist()` should give you the creation property list.